troubadix/plugins/last_modification.py

```diff
@@ -96,6 +99,9 @@ def run(self) -> Iterator[LinterResult]:
         self.new_datetime = correctly_formated_datetime
 
     def fix(self) -> Iterator[LinterResult]:
+        if self.context.nasl_file.suffix == ".inc":
+            return
+
```

Comment:
This code is not necessary. `self.new_file_content` is never set in this case.

Reply:
Thanks, wasn't aware of this detail. I have removed it again with the last push.